src/components/Icon/Icon.vue

```diff
@@ -0,0 +1,80 @@
+<template lang="pug">
+span(:class="className")
+  Text(
+    v-if="accessibilityLabel",
+    as="span",
+  )
+
+  component(
+    :is="source",
+    v-if="sourceType === 'function'",
+    :class="className",
+    :focusable="false",
+    :aria-hidden="true",
+  )
+
+  div(
+    v-if="sourceType === 'placeholder'",
+    :class="styles.placeholder",
+  )
+
+  img(
+    v-if="sourceType === 'external'",
+    alt="",
```

Comment:
?

Reply:
In React version alt attr is not provided

Reply:
![Screenshot from 2023-12-28 09-55-54](https://github.com/ownego/polaris-vue/assets/43405709/7fbae0df-af73-48ef-b929-004b850f2b7d)